.gitmodules

```diff
@@ -1,39 +1,39 @@
 [submodule "submodules/crucible"]
 	path = submodules/crucible
-	url = git@github.com:GaloisInc/crucible.git
+	url = https://github.com/GaloisInc/crucible.git
```

Comment:
Why did all of these change from `git` to `https`?

Reply:
they shouldnt be git right? that means when someone clones they need to ssh auth to pull down the public repos? id assume by default you want them set up as https

Reply:
happy to change them back if we truly want these to be ssh by default, was annoying for me but i could just ssh-agent or whatever instead of what im doing rather than changing these